daemon/cmd/daemon.go

```diff
@@ -1182,7 +1182,7 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup,
 			logfields.V6CiliumHostIP: node.GetIPv6Router(),
 		}).Info("Annotating k8s node")
 
-		err := k8s.AnnotateNode(
+		annotation, err := k8s.AnnotateNode(
```

Comment:
This function is getting out of hand; can we replace most of the parameters with the `node` object?

Reply:
Done.